.circleci/scripts/update_brews.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+set -x
+
+settable_env_vars=(
+  VERSION
+  GIT_BRANCH
+  GIT_COMMIT_HASH
+  REF
+  SHA256
+)
+
+get_tarball() {
+  local -r ref="${1?Ref missing}"
+  curl -L https://api.github.com/repos/fluxninja/aperture/tarball/"${ref}"
```

Comment:
We need to use URL having /archive/ in it as the brew tests are failing with this URL: https://github.com/fluxninja/homebrew-aperture/actions/runs/4087479327/jobs/7048040154#step:7:17